src/SummaryTables.jl

```diff
@@ -44,5 +44,23 @@ include("latex.jl")
 include("html.jl")
 include("docx.jl")
 include("typst.jl")
+include("makie.jl")
+
+function __init__()
+    @static if !isdefined(Base, :get_extension)
+        @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" begin
+            include("../ext/MakieExtension.jl")
+        end
+    end
+
+    Base.Experimental.register_error_hint(MethodError) do io, exc, argtypes, kwargs
+        if exc.f === makie_table!
+            if isempty(methods(exc.f))
+                print(io, "\n$(exc.f) has no methods, yet. Makie has to be loaded for the table extension to be activated. Run `using Makie`, `using CairoMakie`, `using GLMakie` or any other package that also loads Makie.")
+            end
+        end
+    end
+end
```

Comment:
Would it be more straightforward to just define an `::Any` method on `makie_table!` that prints out this message rather than doing an error hint?

Reply:
I had a fallback version with `args...` first and then removed that again for this, I don't remember my rationale though